src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -70,10 +75,172 @@ local function get(url)
     end
 end
 
+local function print_parsed_list(input)
+    local sCodes = string.gmatch(input, "<paste_key>([^\<]+)</paste_key>")
+    local sNames = string.gmatch(input, "<paste_title>([^\<]+)</paste_title>")
+    print("code     | name")
+    local code = sCodes()
+    local name = sNames()
+    while code ~= nil and name ~= nil do
+        print(code.." | "..name)
+        code = sCodes()
+        name = sNames()
+    end
+end
+
 local sCommand = tArgs[1]
-if sCommand == "put" then
+
+local dev_key = "0ec2eb25b6166c0c27a394ae118ad829"
+
+if sCommand == "connect" then
+    write("Username: ")
+    username = read()
+    write("Password: ")
+    password = read("*")
+
+    write("Connecting to pastebin.com... ")
+    local response = http.post(
+    "https://pastebin.com/api/api_login.php",
+    "api_dev_key=" .. dev_key .. "&" ..
+    "api_user_name=" .. textutils.urlEncode(username) .. "&" ..
+    "api_user_password=" .. textutils.urlEncode(password)
```

Comment:
You are saving the password as plain text, personally I would ask for the users password each time theprogram needs it.